File: ama/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/weather/Astronomy.java
package com.android.blawniczak.astroweather.domain.weather;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Astronomy {

    @SerializedName("sunrise")
    @Expose
    public String sunrise;
    @SerializedName("sunset")
    @Expose
    public String sunset;

}

File: paiwjs/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/weather/Astronomy.kt
package com.android.blawniczak.astroweather.domain.weather

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Astronomy {

    @SerializedName("sunrise")
    @Expose
    var sunrise: String? = null
    @SerializedName("sunset")
    @Expose
    var sunset: String? = null

}
